lib/cc/engine/analyzers/python/main.rb

```diff
@@ -12,11 +12,20 @@ module Python
         class Main < CC::Engine::Analyzers::Base
           LANGUAGE = "python"
           DEFAULT_PATHS = ["**/*.py"]
-          DEFAULT_MASS_THRESHOLD = 40
-          BASE_POINTS = 1000
+          DEFAULT_MASS_THRESHOLD = 32
+          BASE_POINTS = 1_500_000
+          POINTS_PER_OVERAGE = 50_000
+
+          def calculate_points(mass)
+            BASE_POINTS + (overage(mass) * POINTS_PER_OVERAGE)
+          end
 
           private
 
+          def overage(mass)
+            mass - mass_threshold
```

Comment:
No, you can also set a custom threshold.

The method is inherited. 
https://github.com/codeclimate/codeclimate-duplication/blob/master/lib/cc/engine/analyzers/analyzer_base.rb#L35